commit a385a52925398e53bedf1a8b30a9a3e002569f27
Author: Kailang Yang <kailang@realtek.com>
Date:   Wed Oct 15 11:20:21 2008 +0200

    ALSA: hda - Add ALC887 support
    
    Added ALC887 support.  It's almost compatible with ALC883/888.
    
    Signed-off-by: Kailang Yang <kailang@realtek.com>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>

Adjusted to apply to Debian's 2.6.26 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.26.orig/sound/pci/hda/patch_realtek.c linux-source-2.6.26/sound/pci/hda/patch_realtek.c
--- linux-source-2.6.26.orig/sound/pci/hda/patch_realtek.c	2009-02-08 23:05:08.000000000 -0700
+++ linux-source-2.6.26/sound/pci/hda/patch_realtek.c	2009-02-08 23:24:29.000000000 -0700
@@ -14360,6 +14360,7 @@ struct hda_codec_preset snd_hda_preset_r
 	{ .id = 0x10ec0885, .rev = 0x100103, .name = "ALC889A",
 	  .patch = patch_alc882 }, /* should be patch_alc883() in future */
 	{ .id = 0x10ec0885, .name = "ALC885", .patch = patch_alc882 },
+	{ .id = 0x10ec0887, .name = "ALC887", .patch = patch_alc883 },
 	{ .id = 0x10ec0888, .name = "ALC888", .patch = patch_alc883 },
 	{ .id = 0x10ec0889, .name = "ALC889", .patch = patch_alc883 },
 	{} /* terminator */
